src/RamlToOpenApiConverter/Extensions/DictionaryExtensions.cs

```diff
@@ -35,6 +35,18 @@ public static T Get<T>(this IDictionary<object, object> d, object key)
             return null;
         }
 
+        public static void Replace(this IDictionary<object, object> d, IDictionary<object, object> c, object key)
```

Comment:
Instead of `d` use `source` or `dictionary`.

And choose a better name for `c`.

Reply:
https://github.com/StefH/RamlToOpenApiConverter/pull/18/commits/db4fe04210ac8f2dc2d776e991e02d2763fed47b